help/md/dnPhyloCTMC.md

```diff
@@ -1,9 +1,45 @@
 ## name
-dnPhyloCTMC
+`dnPhyloCTMC`: Distribution of a phylogenetic continuous-time Markov chain
+
 ## title
+The parameters of a phylogenetic model – a tree topology with branch lengths, a substitution model that describes how observations evolve over the tree, etc. – collectively form a distribution called the _phylogenetic continuous-time Markov chain_.
+
 ## description
+dnPhyloCTMC gives the probability distribution of tip labels on a phylogenetic tree given an phylogenetic continuous-time Markov chain model.
+
 ## details
+
+The likelihood of observed tip labels (specified via a clamped `AbstractHomologousDiscreteCharacterData` object) is computed using Felsenstein's pruning algorithm, with partial likelihoods stored for each branch of the tree. It is automatically outputted in the `Likelihood` column of the `mnFile()` and `mnScreen()` monitors (which can be suppressed with `likelihood = FALSE`).
+
 ## authors
 ## see_also
+- Tutorial on [graphical models](https://revbayes.github.io/tutorials/intro/graph_models)
+
+- Tutorial on [specifying a phylogenetic continuous-time Markov chain](https://revbayes.github.io/tutorials/ctmc/) model
+
 ## example
+
+```rb
+# Create stochastic node with the tip distribution given by the topology `tree` with the 
+# rate matrix parameter q_matrix
+x ~ dnPhyloCTMC(tree = tree, Q = q_matrix)
+
+# Clamp observed characters to the node
+chars <- readDiscreteCharacterData("myData.nex")
+x.clamp(chars)
+
+# Calculate the probability of the observed characters under the given distribution
+x.lnProbability()
```

Comment:
While it's indeed possible for `tree` to be just a topology (e.g., a draw from `dnUniformTopology()` or a similar distribution), the output of `x.lnProbability()` will then be `nan`, because we can't evaluate the likelihood without branch lengths. Perhaps it might be better to provide a slightly expanded, self-contained example:

```
chars <- readDiscreteCharacterData("myData.nex")
taxa = chars.taxa()
# Draw a tree with branch lengths
tree ~ dnUniformTopologyBranchLength(taxa, branchLengthDistribution=dnExp(10.0))
q_matrix <- fnJC(4)
```

and then go on to show how to draw, clamp, and evaluate, as in your current code snippet.

Also, in this context, I would refer to `q_matrix` as an argument rather than a parameter.

Reply:
Good suggestions, thanks; implemented.